README.md

```diff
@@ -13,33 +13,27 @@ Real user monitoring
 
 ### Requirements
 - Node.js >= 6 (```brew install node```)
-- Yarn (```brew install yarn```)
 
 ### Install
 ```sh
 git clone git@github.com:fastly/insights.js.git
 cd insights.js
-yarn
-yarn run build
+npm install
```

Comment:
Why using `npm` instead of `yarn` ?

Reply:
Personally, i've found that since version 6, `npm` has been very fast comparatively. And it's one less dependency for consumers to worry about. It also follows convention with other repos within Fastly.

Reply:
agreed